### PR TITLE
Add 0.11.0 release notes

### PR DESCRIPTION
**Pull request by mukundansundar**
# Description

Added release notes.

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: 

## Checklist
Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

File: docs/release_notes/v0.11.0.md [hunk 1]
```diff
@@ -0,0 +1,489 @@
...
+
+We would like to extend our thanks to all new and existing contributors who helped make this release happen.
+
+**Highlights**
+
+* Support for declarative mechanism to create topic subscription ([Declarative Subscriptions](https://github.com/dapr/docs/blob/master/howto/consume-topic/README.md#declarative-subscriptions))
+* [New access control lists for Service Invocation](https://github.com/dapr/docs/tree/master/concepts/service-invocation#service-access-security)
+* [New scoping for secrets](https://github.com/dapr/docs/blob/master/howto/secrets-scopes/README.md)
+* Support for environment variables as secret store for [development](https://github.com/dapr/docs/blob/master/howto/setup-secret-store/envvar-secret-store.md)
+* New components: RethinkDB binding, [Open Policy agent middleware](https://github.com/dapr/docs/blob/master/howto/policies-with-opa/README.md), Bi-directional Postgres binding
```
Comment (mukundansundar) [inline docs/release_notes/v0.11.0.md:17]: will add links once docs are merged.

**Comment by mukundansundar** [comment]
@yaron2 Please review the release notes and the breaking changes section

File: docs/release_notes/v0.11.0.md [hunk 2]
```diff
@@ -0,0 +1,489 @@
+  
+# Dapr 0.11.0
+
+It's been seven weeks, and We're happy to announce the release of Dapr v0.11.0!
```
Comment (mchmarny) [inline docs/release_notes/v0.11.0.md:4]: We are happy to announce yet another release of Dapr. With this v0.11 release we....

File: docs/release_notes/v0.11.0.md [hunk 3]
```diff
@@ -0,0 +1,489 @@
...
+```
+
+After Dapr 0.11.0 has been installed, perform a rolling restart for your deployments to pick up the new version of the sidecar.
+This can be done with:
+
+```
+kubectl rollout restart deploy/<deployment-name>
+```
+
+## Breaking Changes
```
Comment (mukundansundar) [inline docs/release_notes/v0.11.0.md:408]: Are there any more breaking changes to be added?

File: docs/release_notes/v0.11.0.md [hunk 4]
```diff
@@ -0,0 +1,489 @@
...
+It's been seven weeks, and We're happy to announce the release of Dapr v0.11.0!
+
+With Dapr approaching v1.0, our releases increasingly focus on operational concerns like stability, reliability, and upgradability in production environments.
+As always, community and user requests and feedback are top of mind along with bug fixes.
+
+We would like to extend our thanks to all new and existing contributors who helped make this release happen.
+
+**Highlights**
+
+* Support for declarative mechanism to create topic subscription ([Declarative Subscriptions](https://github.com/dapr/docs/blob/master/howto/consume-topic/README.md#declarative-subscriptions))
```
Comment (mchmarny) [inline docs/release_notes/v0.11.0.md:13]: Declarative topic subscription support

File: docs/release_notes/v0.11.0.md [hunk 5]
```diff
@@ -0,0 +1,489 @@
...
+* [New access control lists for Service Invocation](https://github.com/dapr/docs/tree/master/concepts/service-invocation#service-access-security)
```
Comment (mchmarny) [inline docs/release_notes/v0.11.0.md:14]: Aren't all things in the Highlight section "new"?

File: docs/release_notes/v0.11.0.md [hunk 6]
```diff
@@ -0,0 +1,489 @@
...
+
+### Components
+
+**Refactored** local secret store based on file
+
+**Added** support for environment variables as secret store
+
+**Updated** MQTT publish for multi pub/sub (Thanks @kumaya)
+
+**Fixed** CosmosDB component should be looking for metadata in ExecuteStateTransactionRequest
```
Comment (mchmarny) [inline docs/release_notes/v0.11.0.md:140]: CosmosDB component supports metadata... or similar

File: docs/release_notes/v0.11.0.md [hunk 7]
```diff
@@ -0,0 +1,489 @@
...
+
+**Added** Qos to rabbitMQ binding (Thanks @abserari)
+
+**Added** OPA HTTP middleware (Thanks @RichiCoder1)
+
+**Added** Redis failover feature to redis state store Components (Thanks @abserari)
+
+**Added** code coverage GitHub action
+
+**Added** decode base64 option to output binding  of Azure Blob Storage component (Thanks @mtaghavi2005)
```
Comment (mchmarny) [inline docs/release_notes/v0.11.0.md:152]: option for base64 decoding in Azure Blob Storage output binding component

File: docs/release_notes/v0.11.0.md [hunk 8]
```diff
@@ -0,0 +1,489 @@
...
+
+**Added** case-sensitivity of input parameters (state keys, pubsub topics)
+
+**Added** clarification on use of CloudEvents during publish 
+
+**Updated** Azure Blob Storage binding documentation 
+
+**Refctored** service invocation and state management cocncepts doc
+
+**Updated** KEDA HowTo doc
```
Comment (mchmarny) [inline docs/release_notes/v0.11.0.md:278]: Queue depth autoscaling docs updated for KEDA v2.0-beta

File: docs/release_notes/v0.11.0.md [hunk 9]
```diff
@@ -0,0 +1,489 @@
...
+
+## Upgrading to Dapr 0.11.0
+
+If you're upgrading from an older version of Dapr to 0.11.0, follow the steps here to ensure a smooth upgrade. You know, the one where you don't get red errors on the terminal.. we all hate that, right?
```
Comment (mchmarny) [inline docs/release_notes/v0.11.0.md:282]: To upgrade to this release of Dapr follow...

File: docs/release_notes/v0.11.0.md [hunk 10]
```diff
@@ -0,0 +1,489 @@
...
+
+### Local Machine / Self-hosted
+
+Uninstall Dapr using the CLI you currently have. Note that this will remove the default $HOME/.dapr directory, binaries and all containers dapr_redis, dapr_placement and dapr_zipkin. Linux users need to run sudo if they have installed binary in default path /usr/local/bin/ or have docker command needing sudo.:
```
Comment (mchmarny) [inline docs/release_notes/v0.11.0.md:286]: Uninstall currently Dapr CLI

File: docs/release_notes/v0.11.0.md [hunk 11]
```diff
@@ -0,0 +1,489 @@
...
+
+```bash
+dapr uninstall --all
+```
+
+Next, get the latest CLI following [these](https://github.com/dapr/cli#installing-dapr-cli) instructions, or alternatively download the latest and greatest release from [here](https://github.com/dapr/cli/releases) and put the `dapr` binary in your PATH.
```
Comment (mchmarny) [inline docs/release_notes/v0.11.0.md:292]: Next, follow [these instructions](https://github.com/dapr/cli#installing-dapr-cli) to install the latest CLI version...

File: docs/release_notes/v0.11.0.md [hunk 12]
```diff
@@ -0,0 +1,489 @@
...
+
+Once you have downloaded the CLI, run:
+
+```bash
+dapr init
+```
+
+Wait for the update to finish, and you're good to go!
```
Comment (mchmarny) [inline docs/release_notes/v0.11.0.md:300]: Wait for the update to finish and ensure you are using the latest version of Dapr.

File: docs/release_notes/v0.11.0.md [hunk 13]
```diff
@@ -0,0 +1,489 @@
...
+Make sure you have the right runtime version (0.11.0) with:
+
+```bash
+dapr --version
+```
```
Comment (mchmarny) [inline docs/release_notes/v0.11.0.md:305]: Indicate what this command is expected to return

File: docs/release_notes/v0.11.0.md [hunk 14]
```diff
@@ -0,0 +1,489 @@
...
+
+### Kubernetes
+
+#### Upgrading from 0.10.0
+
+If you previously installed Dapr using Helm, starting with this releases you can now upgrade Dapr to a new version.
```
Comment (mchmarny) [inline docs/release_notes/v0.11.0.md:311]: If you previously installed Dapr using Helm, you can now upgrade your cluster to the newest version of Dapr without the need to uninstall.

File: docs/release_notes/v0.11.0.md [hunk 15]
```diff
@@ -0,0 +1,489 @@
...
+If you installed Dapr using the CLI, go [here](#starting-fresh).
+
+##### 1. Get the latest CLI
+
+Get the latest version of the Dapr CLI as outlined above, and put it in your path.
```
Comment (mchmarny) [inline docs/release_notes/v0.11.0.md:316]: Show how to "add it" to the path

**Review by mchmarny — CHANGES_REQUESTED** [review]
Good start. Left few comments.

File: docs/release_notes/v0.11.0.md (hunk 13 above)
Comment (mukundansundar) [reply docs/release_notes/v0.11.0.md:305]: Will update a day before the release or on Monday.

File: docs/release_notes/v0.11.0.md [hunk 16]
```diff
@@ -0,0 +1,489 @@
...
+
+**Added** error field to Bulk Get API
+
+**Fixed** empty fields in Bulk Get API JSON response
+
+**Changed** perf test payload size
+
+**Added** service invocation REST API Swagger docs (Thanks @dancadana)
+
+**Fixed** actor returns 200 on get if key is not present
```
Comment (tcnghia) [inline docs/release_notes/v0.11.0.md:50]:
```suggestion
**Fixed** actor returns 200 on GET if key is not present
```

File: docs/release_notes/v0.11.0.md [hunk 17]
```diff
@@ -0,0 +1,489 @@
...
+
+**Added** decode base64 option to output binding  of Azure Blob Storage component (Thanks @mtaghavi2005)
+
+**Added** get operation for Azure Blob Storage output binding
+
+**Fixed** broken consistency values for CosmosDB state component
+
+**Fixed** NATS streaming has a NATS logger but not a Dapr logger (Thanks @abserari)
+
+**Fixed** Multi() transaction operations api for redis to use HSET and HDEL
```
Comment (tcnghia) [inline docs/release_notes/v0.11.0.md:160]:
```suggestion
**Fixed** Multi() transaction operations API for redis to use HSET and HDEL
```

File: docs/release_notes/v0.11.0.md (hunk 5 above)
Comment (yaron2) [reply docs/release_notes/v0.11.0.md:14]: We need to make sure the concepts doc links to the How To document. /cc @msfussell @vinayada1

File: docs/release_notes/v0.11.0.md (hunk 1 above)
Comment (yaron2) [inline docs/release_notes/v0.11.0.md:17]: [Open Policy Agent middleware]

File: docs/release_notes/v0.11.0.md [hunk 18]
```diff
@@ -0,0 +1,489 @@
...
+We would like to extend our thanks to all new and existing contributors who helped make this release happen.
+
+**Highlights**
+
+* Support for declarative mechanism to create topic subscription ([Declarative Subscriptions](https://github.com/dapr/docs/blob/master/howto/consume-topic/README.md#declarative-subscriptions))
+* [New access control lists for Service Invocation](https://github.com/dapr/docs/tree/master/concepts/service-invocation#service-access-security)
+* [New scoping for secrets](https://github.com/dapr/docs/blob/master/howto/secrets-scopes/README.md)
+* Support for environment variables as secret store for [development](https://github.com/dapr/docs/blob/master/howto/setup-secret-store/envvar-secret-store.md)
+* New components: RethinkDB binding, [Open Policy agent middleware](https://github.com/dapr/docs/blob/master/howto/policies-with-opa/README.md), Bi-directional Postgres binding
+
```
Comment (yaron2) [inline docs/release_notes/v0.11.0.md:18]: I would add the cross namespace support for service invocation.

File: docs/release_notes/v0.11.0.md [hunk 19]
```diff
@@ -0,0 +1,489 @@
...
+First, update your Helm repos:
+
+```
+helm repo update
+```
+
+Run the following command to upgrade Dapr:
+
+```
+helm upgrade dapr dapr/dapr --version 0.11.0 --namespace dapr-system --reset-values
```
Comment (yaron2) [inline docs/release_notes/v0.11.0.md:330]: The upgrade outlined here will incur downtime. Is there a reason we are not showing the zero downtime upgrade path here?

File: docs/release_notes/v0.11.0.md [hunk 20]
```diff
@@ -0,0 +1,489 @@
...
+
+```
+kubectl rollout restart deploy/<deployment-name>
+```
+
+## Breaking Changes
+
+### Dapr Runtime
+
+#### New ClusterRole for dapr-operator
```
Comment (yaron2) [inline docs/release_notes/v0.11.0.md:412]: Why is this a breaking change?

File: docs/release_notes/v0.11.0.md [hunk 21]
```diff
@@ -0,0 +1,489 @@
...
+
+Removed Cluster Admin permissions and now the dapr-operator has more fine grained [permissions](https://github.com/dapr/dapr/issues/1997). 
+
+#### Serialization for gRPC State Transaction APIs have changed
```
Comment (yaron2) [inline docs/release_notes/v0.11.0.md:416]: Why is this a breaking change?

File: docs/release_notes/v0.11.0.md [hunk 22]
```diff
@@ -0,0 +1,489 @@
...
+```
+Value:    req.Value,
+```
+
+Read more [here](https://github.com/dapr/dapr/issues/2016).
+
+
+### Components Contrib 
+
+#### Serialization for State Transaction APIs have to be handled in component
```
Comment (yaron2) [inline docs/release_notes/v0.11.0.md:435]: Unless I'm missing something, this is not a breaking change as far as the end user is concerned and this entire section can be removed.

File: docs/release_notes/v0.11.0.md [hunk 23]
```diff
@@ -0,0 +1,489 @@
...
+	return bt, err
+}
+
+// in state/redis/redis.go
+bt, _ := utils.Marshal(req.Value, r.json.Marshal)
+```
+
+Read more [here](https://github.com/dapr/dapr/issues/2016).
+
+### Java SDK
```
Comment (yaron2) [inline docs/release_notes/v0.11.0.md:466]: Is this a breaking change or an addition? /cc @artursouza

**Review by yaron2 — CHANGES_REQUESTED** [review]

File: docs/release_notes/v0.11.0.md (hunk 5 above)
Comment (yaron2) [inline docs/release_notes/v0.11.0.md:14]: I would rephrase to: `New access control policies and SPIFFE based identities for Service Invocation`

File: docs/release_notes/v0.11.0.md [hunk 24]
```diff
@@ -0,0 +1,489 @@
...
+
+**Added** service invocation REST API Swagger docs (Thanks @dancadana)
+
+**Fixed** actor returns 200 on get if key is not present
+
+**Added** requests authentication from api-server in sidecar injector
+
+**Fixed** 'host removed' log prints multiple times per second
+
+**Fixed** responsed handling from pub/sub message delivery on gRPC
```
Comment (tcnghia) [inline docs/release_notes/v0.11.0.md:56]:
```suggestion
**Fixed** response handling from pub/sub message delivery on gRPC
```

File: docs/release_notes/v0.11.0.md [hunk 25]
```diff
@@ -0,0 +1,489 @@
...
+
+**Fixed** actor E2E tests intermittent failure
+
+**Added** log for a loaded component in runtime
+
+**Changed** NATS server to NATS streaming for pub/sub (Thanks @abserari)
+
+**Added** declarative mechanism to create topic subscription
+
+**Fixed** sidecar injector leaks private info in logs of kubernetes admission response  **(Needs separate note)**
```
Comment (tcnghia) [inline docs/release_notes/v0.11.0.md:80]:
```suggestion
**Fixed** sidecar injector leaks private info in logs of Kubernetes admission response  **(Needs separate note)**
```

File: docs/release_notes/v0.11.0.md [hunk 26]
```diff
@@ -0,0 +1,489 @@
...
+
+**Fixed** opaque error message on missing "dapr.io/id"
+
+**Added** label to control plane pods in Helm chart
+
+**Fixed** dapr-operator has cluster-admin permissions in kubernetes (Thanks @gunniwho)
```
Comment (tcnghia) [inline docs/release_notes/v0.11.0.md:86]:
```suggestion
**Fixed** dapr-operator has cluster-admin permissions in Kubernetes (Thanks @gunniwho)
```

File: docs/release_notes/v0.11.0.md [hunk 27]
```diff
@@ -0,0 +1,489 @@
...
+
+**Fixed** intermittent E2E test failures for pub/sub
+
+**Added** ability to disable metrics through configuration
+
+**Fixed** serialization issues with transaction upsert gRPC API
+
+**Fixed** Dapr dashboard does not have a Windows image. Force it to linux in Helm chart.
```
Comment (tcnghia) [inline docs/release_notes/v0.11.0.md:94]:
```suggestion
**Fixed** Dapr dashboard does not have a Windows image. Force it to Linux in Helm chart.
```

File: docs/release_notes/v0.11.0.md [hunk 28]
```diff
@@ -0,0 +1,489 @@
...
+
+**Updated** upgrade to go lang version 1.15
+
+**Fixed** logs make it sound like everything is fully running when not
+
+**Fixed** Dapr status command does not show control plane correctly in HA mode
+
+**Fixed** publish command when dashboard is running
+
+**Updated** CLI waits for dapr sidecar before running app without port
```
Comment (tcnghia) [inline docs/release_notes/v0.11.0.md:124]:
```suggestion
**Updated** CLI waits for dapr sidecar before running apps without port
```

File: docs/release_notes/v0.11.0.md [hunk 29]
```diff
@@ -0,0 +1,489 @@
...
+
+**Fixed** CosmosDB component should be looking for metadata in ExecuteStateTransactionRequest
+
+**Added** pub/sub response type
+
+**Added** Qos to rabbitMQ binding (Thanks @abserari)
+
+**Added** OPA HTTP middleware (Thanks @RichiCoder1)
+
+**Added** Redis failover feature to redis state store Components (Thanks @abserari)
```
Comment (tcnghia) [inline docs/release_notes/v0.11.0.md:148]:
```suggestion
**Added** Redis failover feature to Redis state store Components (Thanks @abserari)
```

File: docs/release_notes/v0.11.0.md [hunk 30]
```diff
@@ -0,0 +1,489 @@
...
+```
+
+Read more [here](https://github.com/dapr/dapr/issues/2016).
+
+
+### Components Contrib 
+
+#### Serialization for State Transaction APIs have to be handled in component 
+
+Before v0.11.0, some of the components (eg: redis, postgres) did not handle State Transaction Upsert request value serialization similar to how it was handled in the corresponding `Set` API path. Thus there was a change made in the runtime to convert byte array request values into string which was then passed on to be handled by the component. This caused an issue where properly serialized values were serialized again.
```
Comment (tcnghia) [inline docs/release_notes/v0.11.0.md:437]:
```suggestion
Before v0.11.0, some of the components (eg: Redis, Postgres) did not handle State Transaction Upsert request value serialization similar to how it was handled in the corresponding `Set` API path. Thus there was a change made in the runtime to convert byte array request values into a string which was then passed on to be handled by the component. This caused an issue where properly serialized values were serialized again.
```

File: docs/release_notes/v0.11.0.md [hunk 31]
```diff
@@ -0,0 +1,489 @@
...
+
+```go
+// prior to 0.11.0 in state/redis/redis.go
+b, _ := r.json.Marshal(req.Value)
+```
+
+For v0.11.0, the individual component needs to handle the serialization of the values properly, an example for the redis state transaction upsert operation is:
```
Comment (tcnghia) [inline docs/release_notes/v0.11.0.md:444]:
```suggestion
For v0.11.0, the individual component needs to handle the serialization of the values properly, an example for the Redis state transaction upsert operation is:
```

File: docs/release_notes/v0.11.0.md [hunk 32]
```diff
@@ -0,0 +1,489 @@
...
+
+**Updated** NATS streaming setup and deleted NATS setup (Thanks @abserari)
+
+**Added** case-sensitivity of input parameters (state keys, pubsub topics)
+
+**Added** clarification on use of CloudEvents during publish 
+
+**Updated** Azure Blob Storage binding documentation 
+
+**Refctored** service invocation and state management cocncepts doc
```
Comment (tcnghia) [inline docs/release_notes/v0.11.0.md:276]:
```suggestion
**Refactored** service invocation and state management concepts doc
```

File: docs/release_notes/v0.11.0.md (hunk 10 above)
Comment (tcnghia) [inline docs/release_notes/v0.11.0.md:286]:
```suggestion
Uninstall Dapr using the CLI you currently have. Note that this will remove the default $HOME/.dapr directory, binaries and all containers dapr_redis, dapr_placement and dapr_zipkin. Linux users need to run sudo if they have installed binary in default path /usr/local/bin/ or have docker command needing sudo:
```

File: docs/release_notes/v0.11.0.md [hunk 33]
```diff
@@ -0,0 +1,489 @@
...
+kubectl rollout restart deploy/<deployment-name>
+```
+
+## Breaking Changes
+
+### Dapr Runtime
+
+#### New ClusterRole for dapr-operator
+
+Removed Cluster Admin permissions and now the dapr-operator has more fine grained [permissions](https://github.com/dapr/dapr/issues/1997).
```
Comment (tcnghia) [inline docs/release_notes/v0.11.0.md:414]:
```suggestion
Removed Cluster Admin permissions and now the dapr-operator has more fine-grained [permissions](https://github.com/dapr/dapr/issues/1997). 
```

**Review by tcnghia — CHANGES_REQUESTED** [review]

File: docs/release_notes/v0.11.0.md (hunk 19 above)
Comment (mukundansundar) [reply docs/release_notes/v0.11.0.md:330]: Will there be any issues with the ClusterRole change?

Comment (yaron2) [reply docs/release_notes/v0.11.0.md:330]: There shouldn't be as we are not actually changing any adding or removing permissions.

File: docs/release_notes/v0.11.0.md (hunk 23 above)
Comment (mukundansundar) [reply docs/release_notes/v0.11.0.md:466]: Yes this is a breaking change.

**Review by yaron2 — CHANGES_REQUESTED** [review]
Also add:

`**Fixed** config file to disable mTLS in e2e tests (Thanks @skyao)`

File: docs/release_notes/v0.11.0.md (hunk 2 above)
Comment (orizohar) [reply docs/release_notes/v0.11.0.md:4]: Agree. We can drop the "It's been seven days". Starting from this week we'll include a blog post for each release which can include this kind of tone

File: docs/release_notes/v0.11.0.md (hunk 20 above)
Comment (mukundansundar) [reply docs/release_notes/v0.11.0.md:412]:
```
Error: UPGRADE FAILED: cannot patch "dapr-operator" with kind ClusterRoleBinding: ClusterRoleBinding.rbac.authorization.k8s.io "dapr-operator" is invalid: roleRef: Invalid value: rbac.RoleRef{APIGroup:"rbac.authorization.k8s.io", Kind:"ClusterRole", Name:"dapr-operator-admin"}: cannot change roleRef
```
@yaron2 
Getting this error while following the steps defined in https://github.com/dapr/docs/tree/master/howto/deploy-k8s-prod#upgrading-dapr-with-helm

File: docs/release_notes/v0.11.0.md (hunk 19 above)
Comment (mukundansundar) [reply docs/release_notes/v0.11.0.md:330]:
```
Error: UPGRADE FAILED: cannot patch "dapr-operator" with kind ClusterRoleBinding: ClusterRoleBinding.rbac.authorization.k8s.io "dapr-operator" is invalid: roleRef: Invalid value: rbac.RoleRef{APIGroup:"rbac.authorization.k8s.io", Kind:"ClusterRole", Name:"dapr-operator-admin"}: cannot change roleRef
```
@yaron2 
Getting this error while following the steps defined in https://github.com/dapr/docs/tree/master/howto/deploy-k8s-prod#upgrading-dapr-with-helm

File: docs/release_notes/v0.11.0.md (hunk 22 above)
Comment (mukundansundar) [reply docs/release_notes/v0.11.0.md:435]: This will be a breaking change for gRPC. Since the old code will not work, since the double serialization issue has been fixed in this release.

Comment (yaron2) [reply docs/release_notes/v0.11.0.md:435]: We're talking about breaking changes for end-users, not developers. So what does it mean for an end user that the serialization "have to be handled in component"? Also, the entire piece of code represents internal Dapr workings which should not be of interest to users. IMO we can remove this entire block.

File: docs/release_notes/v0.11.0.md (hunk 20 above)
Comment (yaron2) [reply docs/release_notes/v0.11.0.md:412]: Try adding `--force` to the upgrade command?

File: docs/release_notes/v0.11.0.md (hunk 1 above)
Comment (orizohar) [inline docs/release_notes/v0.11.0.md:17]: Postgres --> PostgreSQL

File: docs/release_notes/v0.11.0.md (hunk 14 above)
Comment (mukundansundar) [reply docs/release_notes/v0.11.0.md:311]: i am not able to upgrade to latest version from 0.10.0. For now removing this section.

Comment (yaron2) [reply docs/release_notes/v0.11.0.md:311]: Have you tried the `--force` upgrade flag I mentioned below? Making the assertion that we cannot upgrade from 0.10 to 0.11 must be back up by the reasons why, which I don't see we currently have.

File: docs/release_notes/v0.11.0.md (hunk 2 above)
Comment (msfussell) [inline docs/release_notes/v0.11.0.md:4]: Suggest shortening to "We're happy to announce the release of Dapr v0.11.0!"

File: docs/release_notes/v0.11.0.md [hunk 34]
```diff
@@ -0,0 +1,489 @@
+  
+# Dapr 0.11.0
+
+It's been seven weeks, and We're happy to announce the release of Dapr v0.11.0!
+
+With Dapr approaching v1.0, our releases increasingly focus on operational concerns like stability, reliability, and upgradability in production environments.
```
Comment (msfussell) [inline docs/release_notes/v0.11.0.md:6]: Let's leave future release schedule out of the release notes. Just start with "The Dapr releases increasingly focus ..."

**Review by AaronCrawfis — APPROVED** [review]
Looks good. In future releases we should also sort changes by category. I suggest Added, Removed, Changed, Updated, then Fixed.

**Comment by yaron2** [comment]
@mukundansundar I see the upgrade process has been removed without an explanation. People will be left to wonder why they cannot upgrade Dapr.

In addition, there are additional review comments on this PR, some of them about the upgrade process.

File: docs/release_notes/v0.11.0.md (hunk 22 above)
Comment (mukundansundar) [reply docs/release_notes/v0.11.0.md:411]: The payload would have been changed. The API will break with old serialized data.

Comment (yaron2) [reply docs/release_notes/v0.11.0.md:411]: /cc @mchmarny @msfussell can you assist in communicating this better to non-Dapr coding users?

**Comment by yaron2** [comment]
@mukundansundar I was able to upgrade successfully and validate with running apps on AKS with the following steps for this release:

### 1. Remove Cluster Role Binding

```
kubectl delete clusterrolebinding dapr-operator
```

### 2. Export certificates
```
dapr mtls export -o ./certs
```

### 3. Upgrade Dapr
```
helm upgrade dapr --namespace=dapr-system --set-string global.tag=0.11.0-rc.3 --set-string global.registry=docker.io/daprio --set-file dapr_sentry.tls.root.certPEM=./certs/ca.crt --set-file dapr_sentry.tls.issuer.certPEM=./certs/issuer.crt --set-file dapr_sentry.tls.issuer.keyPEM=./certs/issuer.key --reset-values ./charts/dapr
```

File: docs/release_notes/v0.11.0.md (hunk 19 above)
Comment (yaron2) [reply docs/release_notes/v0.11.0.md]: See here: https://github.com/dapr/dapr/pull/2119#issuecomment-700328673

**Comment by mukundansundar** [comment]
@yaron2  will add the upgrading section back with the steps described https://github.com/dapr/dapr/pull/2119#issuecomment-700328673

**Comment by yaron2** [comment]
> @yaron2 will add the upgrading section back with the steps described [#2119 (comment)](https://github.com/dapr/dapr/pull/2119#issuecomment-700328673)

Great, thanks,

File: docs/release_notes/v0.11.0.md [hunk 35]
```diff
@@ -0,0 +1,465 @@
...
+
+- **Fixed** actor E2E tests intermittent failure
+
+- **Added** log for a loaded component in runtime
+
+- **Changed** NATS server to NATS streaming for pub/sub (Thanks @abserari)
+
+- **Added** declarative mechanism to create topic subscription
+
+- **Fixed** sidecar injector leaks private info in logs of Kubernetes admission response  **(Needs separate note)**
```
Comment (mchmarny) [inline docs/release_notes/v0.11.0.md:81]: **Fixed** sidecar injector logging admission response (in previous release, under certain condition when running in Kubernetes, this could have included potentially private information).

File: docs/release_notes/v0.11.0.md [hunk 36]
```diff
@@ -0,0 +1,531 @@
+  
+# Dapr 0.11.0
+
+We're happy to announce the release of Dapr v0.11.0!
+
+The Dapr releases are increasingly focusing on operational concerns like stability, reliability, and upgradability in production environments.
```
Comment (mchmarny) [inline docs/release_notes/v0.11.0.md:6]: Consider this: 

> Our Dapr releases are increasingly focused on addressing operational concerns like: security, stability, and reliability. Additionally, given the growing number of Dapr deployments, we have further refined out upgrade process to enable durable production environments. 

Direct user feedback is the primary input into our release planning process. As with every release, we look forward to hearing from you on any friction you may have experienced using Dapr.

File: docs/release_notes/v0.11.0.md [hunk 37]
```diff
@@ -0,0 +1,531 @@
...
+As always, community and user requests and feedback are top of mind along with bug fixes.
```
Comment (mchmarny) [inline docs/release_notes/v0.11.0.md:7]: delete, already mentioned above

File: docs/release_notes/v0.11.0.md [hunk 38]
```diff
@@ -0,0 +1,531 @@
...
+
+We would like to extend our thanks to all new and existing contributors who helped make this release happen.
```
Comment (mchmarny) [inline docs/release_notes/v0.11.0.md:9]: ...all **the** new and existing...

**Review by mchmarny — CHANGES_REQUESTED** [review]
a couple of small copy changes

File: docs/release_notes/v0.11.0.md [hunk 39]
```diff
@@ -0,0 +1,531 @@
...
+
+When deserialized, it would be read as `"saved state"` instead of `saved state`. Similarly for JSON serialized values, it was marshaled once again and then saved.
+
+This has been fixed in v0.11.0 and the double serialization removed. Any code that expects the value in the previous incorrectly serialized format will need some changes to work with v0.11.0.
+
+Read more [here](https://github.com/dapr/dapr/issues/2016).
+
+### Components Contrib 
+
+#### Serialization for State Transaction APIs have to be handled in component
```
Comment (yaron2) [inline docs/release_notes/v0.11.0.md:477]: Can we remove this entire section as it's not a user facing change?

File: docs/release_notes/v0.11.0.md [hunk 40]
```diff
@@ -0,0 +1,531 @@
...
+\"saved state\"
+```
+
+When deserialized, it would be read as `"saved state"` instead of `saved state`. Similarly for JSON serialized values, it was marshaled once again and then saved.
+
+This has been fixed in v0.11.0 and the double serialization removed. Any code that expects the value in the previous incorrectly serialized format will need some changes to work with v0.11.0.
+
+Read more [here](https://github.com/dapr/dapr/issues/2016).
+
+### Components Contrib
```
Comment (yaron2) [inline docs/release_notes/v0.11.0.md:475]: Need to add another breaking change: In Pub/Sub, the implementation for NATS changed from NATS server to NATS streaming, and everyone using NATS should now use the NATS streaming as explained [here](https://github.com/dapr/docs/blob/master/howto/setup-pub-sub-message-broker/setup-nats-streaming.md).

**Review by yaron2 — CHANGES_REQUESTED** [review]

File: docs/release_notes/v0.11.0.md (hunk 39 above)
Comment (msfussell) [reply docs/release_notes/v0.11.0.md:477]: We we remove this implementation details and point to the detailed issue "Read more [here](https://github.com/dapr/dapr/issues/2016). " so that people can find out more as a component developer

**Review by artursouza — APPROVED** [review]

**Review by yaron2 — APPROVED** [review]
lgtm

**Review by tcnghia — APPROVED** [review]

**Review by mchmarny — APPROVED** [review]
/lgtm
